MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -58,6 +59,7 @@ bool LayeredVolume::createRasterLayers(const MeshLib::Mesh &mesh,
 	this->_minimum_thickness = minimum_thickness;
 	_nodes = MeshLib::copyNodeVector(bottom->getNodes());
 	_elements = MeshLib::copyElementVector(bottom->getElements(), _nodes);
+	_materials.insert(_materials.begin(), _elements.size(), 0); 
```

Comment:
I have an empty vector and, say, 100 elements. What is the difference between inserting 100 x "0" and filling it with 100 x "0" ?